src/_posts/platform/app/2000-01-01-send-signal.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Send a signal to a container
+nav: Send Signal
+modified_at: 2023-01-12 00:00:00
+tags: app signal
+---
+
+## Introduction
+
+Signals are standardized messages sent to a running program to trigger specific behavior.
+
+On the Scalingo platform, sending signals is a useful way to trigger events while being the owner or collaborator of an application. This can be particularly useful for debugging purposes.
+
+{% note %}
+  The platform handles only `SIGUSR1` and `SIGUSR2`.
+{% endnote %}
+
+## General purpose of the feature
+
+As said before, implementing a signal handler can be an easy and powerful way to debug applications.
+
+At Scalingo we often use signals handler to:
+
+- Dump heap allocations
+- Dump all stack traces
+- Print the number of Goroutines
+- Print the memory consumption
+- etc…
+
+## How to send a signal to an application ?
+
+To send a signal to an application, multiple possibilities are available:
+
+- The CLI provides a `send-signal` command.
+
+```sh
+$ scalingo -a my-app ps
++-------+---------+-------------+------+---------------------+
+| NAME  | STATUS  |   COMMAND   | SIZE |     CREATED AT      |
++-------+---------+-------------+------+---------------------+
+| web-1 | running | sample-test | M    | 2023/01/05 09:24:33 |
+| web-2 | running | sample-test | M    | 2023/01/05 09:24:33 |
++-------+---------+-------------+------+---------------------+
+
+$ scalingo -a my-app send-signal --signal SIGUSR1 web-1
+-----> Sent signal 'SIGUSR1' to 'web-1' container.
+```
+
+- An API endpoint is available to send a signal. More information about the endpoint [here](https://developers.scalingo.com/apps#send-signal-to-a-container).
+
+## Who catches the signal ?
```

Comment:
I think that in English, "who" refers to a person. But in this case you are not talking about a person.